store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/FederatedStoreTest.java

```diff
@@ -335,15 +335,15 @@ public void shouldUpdateSchemaWhenNewGraphIsAdded() throws Exception {
     public void shouldUpdateSchemaWhenNewGraphIsRemoved() throws Exception {
         // Given
         addGraphWithPaths(ACC_ID_1, propertiesAlt, blankUserContext, SCHEMA_ENTITY_BASIC_JSON);
-        final Schema was = store.getSchema(new Context(blankUser));
+        final Schema was = store.getSchema(new Context(blankUser), true);
```

Comment:
If there was a `getSchema` added which took no args and defaulted to true (or false; whichever was default behaviour), then there would be a lot less unnecessary changes

Reply:
I was considering adding one to do exactly that, but I believe this was done deliberately by @GCHQDev404 in order to ensure clarity?